medusa/tv/series.py

```diff
@@ -1554,14 +1555,21 @@ def load_imdb_info(self):
                       {'id': self.indexerid, 'imdb_id': self.imdb_id})
             return
 
+        tmdb_id = self.externals.get('tmdb_id')
+        if tmdb_id:
+            country_code = Tmdb()._get_shows_countries(tmdb_id)
+            countries = '|'
+            for country in country_code.split('|'):
+                countries = countries.join(subtitles.name_from_code(country))
```

Comment:
@p0psicles is this efficient? it will keep joining already joined countries

I added generator here but got removed